src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -36,60 +47,322 @@ $pf-c-tree-view--MaxNesting: 10;
   --pf-c-tree-view__list-item--m-expanded__node-toggle-icon--Rotate: 90deg;
 
-  // text
+  // Text
   --pf-c-tree-view__node-text--max-lines: 1;
 
-  // action
+  // Title
+  --pf-c-tree-view__node-title--FontWeight: var(--pf-global--FontWeight--bold);
+
+  // Action
   --pf-c-tree-view__action--MarginLeft: var(--pf-global--spacer--md);
   --pf-c-tree-view__action--focus--BackgroundColor: var(--pf-global--BackgroundColor--200);
   --pf-c-tree-view__action--Color: var(--pf-global--icon--Color--light);
   --pf-c-tree-view__action--hover--Color: var(--pf-global--icon--Color--dark);
   --pf-c-tree-view__action--focus--Color: var(--pf-global--icon--Color--dark);
 
+  // Guides
+  // ================================================================== //
+  --pf-c-tree-view--m-guides--border--Left: var(--pf-c-tree-view--m-guides--base-border--Left); // starting value, this gets updated for each nesting level
+  --pf-c-tree-view--m-guides--border--ZIndex: var(--pf-c-tree-view--m-guides--base-border--ZIndex);
+
+  // Base
+  --pf-c-tree-view--m-guides--base-border--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-tree-view--m-guides--base-border--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tree-view--m-guides--base-border--Left: calc(var(--pf-c-tree-view__node--PaddingLeft) - var(--pf-c-tree-view--m-guides__list-node--base--BorderWidth));
+  --pf-c-tree-view--m-guides--base-border--Left--offset: calc(var(--pf-c-tree-view__list-item__list-item__node-toggle--Left) + var(--pf-c-tree-view__node-toggle-icon--MinWidth) / 2); // based on toggle positioning
+  --pf-c-tree-view--m-guides--base-border--ZIndex: var(--pf-global--ZIndex--sm);
+  --pf-c-tree-view--m-guides__list-node--base--BorderWidth: var(--pf-global--spacer--lg);
+
+  // List item
+  --pf-c-tree-view--m-guides__list-item--before--Top: 0;
+  --pf-c-tree-view--m-guides__list-item--before--Width: var(--pf-c-tree-view--m-guides--base-border--BorderWidth);
+  --pf-c-tree-view--m-guides__list-item--before--Height: 100%;
+  --pf-c-tree-view--m-guides__list-item--before--BackgroundColor: var(--pf-c-tree-view--m-guides--base-border--BorderColor);
+  --pf-c-tree-view--m-guides__list-item--last-child--before--Top: var(--pf-c-tree-view--m-guides__node--before--Top);
+  --pf-c-tree-view--m-guides__list-item--last-child--before--Height: var(--pf-c-tree-view--m-guides__list-item--last-child--before--Top);
+
+  // Node
+  --pf-c-tree-view--m-guides__node--before--Width: #{pf-size-prem(16px)};
+  --pf-c-tree-view--m-guides__node--before--Height: var(--pf-c-tree-view--m-guides--base-border--BorderWidth);
+  --pf-c-tree-view--m-guides__node--before--Top: #{pf-size-prem(18px)};
+  --pf-c-tree-view--m-guides__node--before--BackgroundColor: var(--pf-c-tree-view--m-guides--base-border--BorderColor);
+
+  // Compact
+  // ================================================================== //
+  // Base
+  --pf-c-tree-view--m-compact--base-border--Left--offset: var(--pf-global--spacer--md);
```

Comment:
Curious why some are `--base-border--Left--offset` and others are `--indent--base`.

Reply:
Indent refers to the node padding while offset refers to the guide positioning. They separate because they change relatively as nesting increases past 2 levels deep and need to be calculated differently.